test/groovy/TransportRequestUploadFileTest.groovy

```diff
@@ -136,29 +139,26 @@ public class TransportRequestUploadFileTest extends BasePiperTest {
     @Test
     public void uploadFileToTransportRequestSOLMANFailureTest() {
 
-        ChangeManagement cm = new ChangeManagement(nullScript) {
-            void uploadFileToTransportRequestSOLMAN(
-                                              Map docker,
-                                              String changeId,
-                                              String transportRequestId,
-                                              String applicationId,
-                                              String filePath,
-                                              String endpoint,
-                                              String credentialsId,
-                                              String cmclientOpts) {
-                throw new ChangeManagementException('Exception message')
-            }
-        }
-
         thrown.expect(AbortException)
         thrown.expectMessage("Exception message")
 
+        helper.registerAllowedMethod( 'piperExecuteBin', [Map, String, String, List], { 
```

Comment:
picky, but:
```suggestion
        helper.registerAllowedMethod('piperExecuteBin', [Map, String, String, List], { 
```
Maybe you could run a formatter.

Reply:
I'll do.
I didn't do this, cause it changes the whole file so that the review would be almost impossible.